bundler/lib/bundler/definition.rb

```diff
@@ -571,7 +571,7 @@ def resolution_packages
       @resolution_packages ||= begin
         last_resolve = converge_locked_specs
         remove_invalid_platforms!
-        packages = Resolver::Base.new(source_requirements, expanded_dependencies, last_resolve, @platforms, locked_specs: @originally_locked_specs, unlock: @gems_to_unlock, prerelease: gem_version_promoter.pre?)
+        packages = Resolver::Base.new(source_requirements, expanded_dependencies, last_resolve, @platforms, locked_specs: @originally_locked_specs, unlock: @gems_to_unlock, prerelease: gem_version_promoter.pre?, prefer_local: @prefer_local)
```

Comment:
Not sure how `prefer_local` is received in the `Base`'s initializer. Does it go to the `options` hash?

https://github.com/rubygems/rubygems/blob/636af9667c1253371cc43c46d15609ed18eed74a/bundler/lib/bundler/resolver/base.rb#L10

Reply:
Yes, it goes into the `options` hash.